Bio/PDB/__init__.py

```diff
@@ -34,6 +35,8 @@
 # Get a Structure object from a PDB file
 from .PDBParser import PDBParser
 
+if importlib.util.find_spec("msgpack"):
+    from .binary_cif import BinaryCIFParser
```

Comment:
I wouldn't import this here at all - let the user do it on demand as per ``Bio/PDB/mmtf/__init__.py``